test/test_compile.py

```diff
@@ -14,11 +14,40 @@
 # a commercial license, send an email to license@arduino.cc.
 import json
 import os
+
 import pytest
 
 from .common import running_on_ci
 
 
+def test_sketch_new(run_command):
```

Comment:
we should probably create a new module `test_sketch.py` for this

Reply:
Module created. Additionally, I applyied a small modification to the `run_command` fixture in order to have a temp working dir as a sandbox for the sketch creation e2e tests